dss/dynamodb/__init__.py

```diff
@@ -3,53 +3,54 @@
 from dss.util.aws.clients import dynamodb as db  # type: ignore
 
 
-def put_item(table: str, value: str, hash_key: str, sort_key: str):
+def format_item(value, hash_key, sort_key):
+    item = {'hash_key': {'S': hash_key}}
+    if value:
+        item['body'] = {'S': value}
+    if sort_key:
+        item['sort_key'] = {'S': sort_key}
+    return item
+
+
+def put_item(table: str, value: str, hash_key: str, sort_key: str=None, overwrite: str=None):
     """
     Put an item into a dynamoDB table.
 
+    Will determine the type of db this is being called on by the number of keys provided (omit
+    sort_key to PUT into a db with only 1 primary key).
+
     :param table: Name of the table in AWS.
     :param str value: Value stored by the two primary keys.
     :param str hash_key: 1st primary key that can be used to fetch associated sort_keys and values.
     :param str sort_key: 2nd primary key, used with hash_key to fetch a specific value.
+                         Note: If not specified, this will PUT only 1 key (hash_key) and 1 value.
+    :param str overwrite: Don't overwrite if this parameter exists.  For example, setting this
```

Comment:
What happens if `overwrite` exists? e.g., is an exception thrown?

Reply:
That's a good question.  I may have added this too early since this was from the collections portion.  Will take this out and test it when I add it with the collections portion.